package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tatumio/tatum",
-  "version": "1.6.15",
+  "version": "1.6.16",
```

Comment:
1.7.0, because you did incompatible changes by removing testnet from ETH methods